ci-operator/step-registry/telcov10n/functional/cnf-core/telcov10n-functional-cnf-core-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: telcov10n-functional-cnf-core
+  steps:
+    pre:
+    - ref: telcov10n-functional-ocp-baremetal-ipi-deployment
+    - ref: telcov10n-functional-ocp-olm-setup
+    - ref: telcov10n-functional-ocp-cnf-core-tests-prepare
```

Comment:
Are there any jobs that should run in the "post" stanza? 
for example: reporting